toolkit/tools/scheduler/schedulerutils/preparerequest.go

```diff
@@ -66,7 +66,7 @@ func canUseCacheForNode(pkgGraph *pkggraph.PkgGraph, node *pkggraph.PkgNode, pac
 
 	// Check if the node corresponds to an entry in packagesToRebuild
 	specName := strings.TrimSuffix(filepath.Base(node.SpecPath), specSuffix)
-	canUseCache = (sliceutils.Find(packagesToRebuild, specName) == -1)
+	canUseCache = (sliceutils.Find(packagesToRebuild, specName, sliceutils.StringMatch) == -1)
```

Comment:
```suggestion
	canUseCache = (sliceutils.Find(packagesToRebuild, specName, sliceutils.DoStringsMatch) == -1)
```

Reply:
Statying with the current one. See: previous comment.